index.html

```diff
@@ -115,10 +200,32 @@ <h2 class="post">Multi-Post <br>
                 <button type="button" class="button">See Project</button>
             </div>
             </div>
-         
+            <div class="block5">
+                <h2 class="post">Uber Navigation </h2>
+                    <div class="frame">
+                        <img src="images/frame3.png" alt="tabs">
+                    </div>
+            <p class="content-text-px">A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
+            <div class="tags">
+                <div class="tags1">
+                <ul>
+                    <li><a href="#" class="tag-item1">html</a></li>
+                    <li><a href="#" class="tag-item1">Ruby on Rails</a></li>
+                    <li><a href="#" class="tag-item1">css</a></li>
+                    <li><a href="#" class="tag-item1">javascript</a></li>
+                </ul>
+            </div></div>
+            <div class="action">
+                <button type="button" class="button">See Project</button>
+            </div>
+            </div>
+         </div>
+         </div>
         </div>
         <div>
             <div class="content-x">
+                <div class="content-x-y">
+                    <div id="about1">
```

Comment:
Kindly also fix the about Section on mobile too 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/188266236-9c1d523e-8cbf-4bbd-af5d-0c3ba3c0fe67.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/188266257-99330d87-641f-432f-b137-66ff795a7305.png)